src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -26,8 +26,12 @@ public class LineService {
     @Transactional
     public LineResponse saveLine(LineRequest lineRequest) {
 
-        Line line = lineRepository.save(new Line(lineRequest.getName(), lineRequest.getColor(), new LineSections()));
-        LineSection lineSection = new LineSection(line, stationRepository.findByIdOrThrow(lineRequest.getUpStationId()), stationRepository.findByIdOrThrow(lineRequest.getDownStationId()), lineRequest.getDistance());
+        Line line = lineRepository.save(
+            new Line(lineRequest.getName(), lineRequest.getColor(), new LineSections()));
+        LineSection lineSection = new LineSection(line,
+            stationRepository.findByIdOrThrow(lineRequest.getUpStationId()),
+            stationRepository.findByIdOrThrow(lineRequest.getDownStationId()),
+            lineRequest.getDistance());
```

Comment:
Nit: 개인적으로는 모든 파라미터를 줄바꿈으로 처리하는 게 가독성이 더 낫지 않을까란 의견 드려봅니다.
```suggestion
        LineSection lineSection = new LineSection(
            line,
            stationRepository.findByIdOrThrow(lineRequest.getUpStationId()),
            stationRepository.findByIdOrThrow(lineRequest.getDownStationId()),
            lineRequest.getDistance()
        );
```